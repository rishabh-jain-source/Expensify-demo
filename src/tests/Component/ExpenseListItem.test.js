// import React from 'react'
// import { shallow } from 'enzyme'
// import { ExpenseListItem } from '../../components/ExpenseListItem'
// import toJSON from 'enzyme-to-json'

// const expenses = [{
//     id: 1,
//     description: 'rent',
//     note: '',
//     amount: 100,
//     createdAt: 0
// }]

// test("show item", () => {
//     const wrapper = shallow(<ExpenseListItem {...expenses[0]} />)
//     expect(toJSON(wrapper)).toMatchSnapshot();
// })
test('add', () => {
    expect(1).toBe(1);
})