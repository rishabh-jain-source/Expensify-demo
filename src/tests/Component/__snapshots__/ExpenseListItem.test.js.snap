// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`show item 1`] = `
<div>
  <Link
    to="/EditExpense/1"
  >
    rent
  </Link>
  <p>
    $100.00
    -
    January 1st,1970
  </p>
</div>
`;
